src/pages/Address/index.tsx

```diff
@@ -124,12 +124,14 @@ export const Address = () => {
           hash={address}
           specialAddress={addressInfoQuery.data?.isSpecial ? addressInfoQuery.data.specialAddress : ''}
           showDASInfoOnHeader={addressInfoQuery.data?.addressHash ?? false}
-        >
-          <AddressTitleOverview address={addressInfoQuery.data ?? defaultAddressInfo} />
-        </AddressHashCard>
+        />
 
         <QueryResult query={addressInfoQuery} delayLoading>
-          {data => <AddressAssetComp address={data} />}
+          {data => (
+            <AddressAssetComp address={data}>
+              <AddressTitleOverview address={addressInfoQuery.data ?? defaultAddressInfo} />
```

Comment:
The current design appears unnecessary. Both `AddressAssetComp` and `AddressTitleOverview` are expected to be components used only once, so the split was made purely for the sake of readability rather than code reuse. Writing the code in this way only reduces readability and would confuse future maintainers who are unaware of the reasoning behind this split.

I recommend moving `AddressTitleOverview` directly into `AddressAssetComp` and renaming `AddressAssetComp` if necessary (although it appears that the current name is still suitable).